js/views/modals/orderDetail/contractTab/Contract.js

```diff
@@ -1,5 +1,8 @@
+import $ from 'jquery';
+import { clipboard } from 'electron';
 import renderjson from '../../../../lib/renderjson';
 import BaseVw from '../../../baseVw';
```

Comment:
Could we also put the BaseVw below the util mods?

I know you just followed the already out-of-order approach in place.